common/params_pxd.pxd

```diff
@@ -0,0 +1,17 @@
+from libcpp.string cimport string
+from libcpp cimport bool
+
+cdef extern from "selfdrive/common/params.cc":
+  pass
+
+cdef extern from "selfdrive/common/util.c":
+  pass
+
+# Declare the class with cdef
+cdef extern from "selfdrive/common/params.h":
+  cdef cppclass Params:
+    Params(bool)
+    Params(string)
+    string get(string, bool) nogil
+    int delete_db_value(const char*);
+    int write_db_value(string, string);
```

Comment:
```suggestion
    int delete_db_value(const char*)
    int write_db_value(string, string)
```

Reply:
why does delete take a const char and everything else uses a string?

Reply:
Fixed. Think we should move everything in c++ over to the new std::string versions at some point and retire the pointer based ones.